src/burn/engine/apply.cpp

```diff
@@ -452,6 +454,9 @@ extern "C" HRESULT ApplyCache(
     hr = UserExperienceInterpretExecuteResult(pUX, FALSE, MB_OKCANCEL, nResult);
     ExitOnRootFailure(hr, "UX aborted cache.");
 
+    // When in cache mode, we want to download in the background (if the download mechanism supports it).
+    BOOL fPreferBackgroundDownload = (BOOTSTRAPPER_ACTION_CACHE == pPlan->action);
```

Comment:
Background downloads removed. Originally it was to support async execution anyway, so probably didn't fit well with this change anyway.